pkg/reconciler/cronjobsource/cronjobsource.go

```diff
@@ -276,7 +287,7 @@ func (r *Reconciler) reconcileEventType(ctx context.Context, src *v1alpha1.CronJ
 
 	// Only create EventTypes for Broker sinks. But if there is an EventType and the src has a non-Broker sink
 	// (possibly because it was updated), then we need to delete it.
-	if src.Spec.Sink.Kind != "Broker" {
+	if src.Spec.Sink.Ref == nil || src.Spec.Sink.Ref.Kind != "Broker" {
```

Comment:
Do we need to support the deprecated Broker sink too?

Reply:
Fixed